### PR TITLE
ECL-510: Welsh translations

### PR DESCRIPTION
**Pull request by TheDanielDoyle**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -7,45 +55,95 @@ hasEclReference.No = Na
...
+
 eclRegistrationDate.title = Eich dyddiad cofrestru ECL
 eclRegistrationDate.heading = Eich dyddiad cofrestru ECL
 eclRegistrationDate.p1 = Rhowch wybod i ni’r dyddiad y gwnaethoch gofrestru ar gyfer ECL. Gallwch ddod o hyd i hwn yn y llythyr a gawsoch gan CThEF, neu yn yr e-bost cadarnhau a gawsoch pan wnaethoch gofrestru.
 eclRegistrationDate.hint = Er enghraifft, 13 9 2023
 eclRegistrationDate.error.notWithinRange = Mae’n rhaid i’r dyddiad fod rhwng 1 Ebrill 2022 a heddiw
-detailsDoNotMatch.title = Nid yw’ch manylion yn cyd-fynd â’n cofnodion
-detailsDoNotMatch.heading = Nid yw’ch manylion yn cyd-fynd â’n cofnodion
-detailsDoNotMatch.p1 = Nid yw’r manylion yr ydych wedi’u rhoi yn cyd-fynd â’n cofnodion.
-detailsDoNotMatch.p2 = Edrychwch ar y llythyr a gawsoch gan CThEF, neu’r e-bost a gawsoch pan wnaethoch gofrestru ar gyfer yr Ardoll Troseddau Economaidd (ECL).
```
Comment (paddynski-opencast) [inline conf/messages.cy:32]: Why is this removed?

Comment (TheDanielDoyle) [reply conf/messages.cy:32]: Whoever did the previous Welsh translations duplicated those message keys. Seee lines 39 to 42. You can't have duplicate keys in a messages file.

**Review by paddynski-opencast — APPROVED** [review]
